lib/babe/verify.go

```diff
@@ -331,13 +330,49 @@ func (b *verifier) verifyAuthorshipRight(header *types.Header) error {
 	if err != nil {
 		return fmt.Errorf("could not verify block equivocation: %w", err)
 	}
+
 	if equivocated {
 		return fmt.Errorf("%w for block header %s", ErrProducerEquivocated, header.Hash())
 	}
 
 	return nil
 }
 
+func (b *verifier) submitAndReportEquivocation(
+	slot uint64, authorityIndex uint32, firstHeader, secondHeader types.Header) error {
+
+	// TODO: Check if it is initial sync
```

Comment:
I saw you created issue for this, maybe include the number in this comment

Reply:
can you link the created issue?

Reply:
https://github.com/ChainSafe/gossamer/issues/3004